domainlab/arg_parser.py

```diff
@@ -295,7 +295,8 @@ def mk_parser_main():
         "--loglevel", type=str, default="DEBUG", help="sets the loglevel of the logger"
     )
     arg_group_task.add_argument(
-        "--shuffle_off", action="store_false", default=True, help="shuffle of the training/testing/validation dataset"
+        "--shuffling", action="store_true", default=False,
```

Comment:
@sidulova doesn't this turn shuffle off by default (`default=False` for `shuffling`), and shuffles only when explicitly passed `--shuffling`? It needs to be shuffling by default. I think the `shuffle_off` flag made more sense, but it needs to be defaulting to False (i.e., shuffle_off=False by default, meaning that shuffle is on).

Reply:
I based on ```--nocu``` parameter (https://github.com/marrlab/DomainLab/blob/801b12b3d1e088ea242fec2fc49bc98b5851cbe1/domainlab/arg_parser.py#L41), when it is called then it should be ```False```, otherwise it is always assumed to be ```True```